src/atoms/Button/__tests__/Button.native.test.js

```diff
@@ -1,26 +1,103 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react-native';
+import { ThemeProvider } from 'styled-components/native';
 
+import theme from '../../../tokens/theme';
 import Button from '../index';
 
+const renderWithTheme = (children) =>
+  render(<ThemeProvider theme={theme}>{children}</ThemeProvider>);
+
 describe('Renders <Button /> correctly', () => {
-  it('snapshot testing', () => {
-    const { container } = render(<Button onClick={() => {}}>{'Click Me'}</Button>);
+  it('Renders a Primary <Button />', () => {
+    const { container } = renderWithTheme(<Button icon="info">Click Me</Button>);
```

Comment:
As a standard practice, can we keep all text in {" "} ? Github understands it as text better.
```suggestion
    const { container } = renderWithTheme(<Button icon="info">{"Click Me"}</Button>);
```

Reply:
What would be the benefit of doing this?